src/components/Footer/Footer.tsx

```diff
@@ -3,8 +3,6 @@ import { Facebook, Instagram, Twitter, GitHub } from '@mui/icons-material';
 import './footer.css';
 function Footer() {
   return (
-    <div className="footerContainer">
-      <div className="topFooter"></div>
```

Comment:
I guess I should have been explicit but I guess I wasn't expecting this to be taken out in this PR, was planning on adding another layer to the footer once the positioning was figured out